<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QMainWindow" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PictureIcon">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid black</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Icons/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="connectionLabelStatus">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
    </property>
    <property name="text">
     <string>User Info</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="noteSpaceLogo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Icons/notes_mainLabel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>311</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
	background: color white;
}</string>
       </property>
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="userNotesList"/>
     </item>
     <item>
      <widget class="QPushButton" name="listOpenButton">
       <property name="text">
        <string>Open Note</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userNameLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Major: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="schoolLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>School: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="majorLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>PictureIcon</zorder>
   <zorder>connectionLabelStatus</zorder>
   <zorder>infoLabel</zorder>
   <zorder>noteSpaceLogo</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_Note"/>
    <addaction name="actionAdd_Note"/>
    <addaction name="actionView_Notes"/>
    <addaction name="actionLog_Out"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_Note"/>
   <addaction name="actionAdd_Note"/>
   <addaction name="actionView_Notes"/>
   <addaction name="actionLog_Out"/>
  </widget>
  <action name="actionAdd_Note">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/Icons/new-file-icon.png</normaloff>
     <normalon>Icons/Document-Write-icon.png</normalon>:/pics/Icons/new-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Add Note</string>
   </property>
  </action>
  <action name="actionView_Notes">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/Icons/search_files.png</normaloff>:/pics/Icons/search_files.png</iconset>
   </property>
   <property name="text">
    <string>View Notes</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/logout-icon.png</normaloff>:/Icons/logout-icon.png</iconset>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionCreate_Note">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/Document-Write-icon.png</normaloff>:/Icons/Document-Write-icon.png</iconset>
   </property>
   <property name="text">
    <string>Create Note</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
